### PR TITLE
Report the relative error for single queries

### PR DESCRIPTION
**Pull request by Cecca**

### CODE REVIEW

File: ann_benchmarks/plotting/utils.py [hunk 1]
```diff
@@ -178,6 +178,7 @@ def compute_metrics(dataset, res, metric_1, metric_2, recompute=False):
     "glove-2m-300-angular": "GLOVE-2M" ,
     "gnews-300-angular": "GNEWS" ,
     "glove-100-angular": "GLOVE" ,
+    "glove-100-angular-10": "GLOVE(k=10)" ,
```
Comment (maumueller) [inline ann_benchmarks/plotting/utils.py:181]: which version is that? Should this be included in the PR?

File: ann_benchmarks/plotting/metrics.py [hunk 2]
```diff
@@ -73,6 +73,31 @@ def rel(dataset_distances, run_distances, metrics):
...
+        rel_group.attrs['std'] = np.std(relative_errors)
+        percentiles = [5,25,50,75,95]
+        percentile_values = np.percentile(relative_errors, percentiles)
+        for p, v in zip(percentiles, percentile_values):
+            rel_group.attrs['perc-' + str(p)] = v
+        rel_group['relative_errors'] = relative_errors
+    else:
+        print("Found cached result")
+    return metrics['rel']
+
```
Comment (maumueller) [inline ann_benchmarks/plotting/metrics.py:100]: It seems to me that we duplicate a lot of code with these percentile calculations, at some point we should refactor.

**Review by maumueller — COMMENTED** [review]
Looks good, could you double check that the glove version should really be included in master?

**Comment by Cecca** [comment]
All of this is superseded by #14 actually: we no longer need all the duplication for the percentiles (since we can easily compute them from the db), and the datasets should be just the correct ones. I think we can close this pull request without merging

**Comment by Cecca** [comment]
Obsoleted by #14, not merging

**Pull request closed without merging**